proof/refine/ARM/Finalise_R.thy

```diff
@@ -3163,19 +3150,6 @@ lemma finaliseCapTrue_standin_bound_tcb_at':
   apply (case_tac cap; simp add: finaliseCapTrue_standin_def isCap_simps)
   by wpsimp
 
-lemma capDeleteOne_bound_tcb_at':
-  "\<lbrace>bound_tcb_at' P tptr and cte_wp_at' (isReplyCap \<circ> cteCap) callerCap\<rbrace>
-      cteDeleteOne callerCap \<lbrace>\<lambda>rv. bound_tcb_at' P tptr\<rbrace>"
```

Comment:
Is there a more general version of this lemma now? We clearly don't use it since it can be removed, but it feels like one which might be useful to have available.

Reply:
There isn't a more general version of this lemma. I can add it back in, though for some reason the proof requires a slight update (I think part of the reason I removed it was because this proof broke and I thought that if it wasn't used then it might not be worth fixing)

Reply:
I'm ok with not bothering. I can imagine it being useful but it's probably not worth fixing now with it not being used.

Reply:
Whoops, sorry, I see you've already done it.